googletest/include/gtest/gtest-matchers.h

```diff
@@ -474,13 +474,13 @@ class PolymorphicMatcher {
    public:
     explicit MonomorphicImpl(const Impl& impl) : impl_(impl) {}
 
-    virtual void DescribeTo(::std::ostream* os) const { impl_.DescribeTo(os); }
+    virtual void DescribeTo(::std::ostream* os) const override { impl_.DescribeTo(os); }
```

Comment:
Here and everywhere else, remove `virtual`, as per the google style guide:
https://google.github.io/styleguide/cppguide.html#Inheritance

Thanks!

Reply:
Done in #2475 - can't push freely to the source branch for this PR.